corehq/apps/fixtures/upload/workbook.py

```diff
@@ -263,17 +264,16 @@ def _get_field_is_indexed(prop_key):
                     raise FixtureUploadError([error_message])
             return is_indexed
 
-        def is_number(text):
-            text = str(text)
+        def is_valid_field_name(text):
             try:
-                float(text)
+                Element(text)
                 return True
-            except ValueError:
+            except (ValueError, TypeError):
```

Comment:
Do you know if it's still possible for a `ValueError` to be thrown here?

Reply:
Checked locally and see that it is still possible, where `ValueError` is thrown for an invalid tag name and `TypeError` for an invalid argument type (like an integer).